config/fcos/v1_6_exp/translate.go

```diff
@@ -367,3 +375,68 @@ func buildGrubConfig(gb Grub) string {
 	superUserCmd := fmt.Sprintf("set superusers=\"%s\"\n", strings.Join(allUsers, " "))
 	return "# Generated by Butane\n\n" + superUserCmd + strings.Join(cmds, "\n") + "\n"
 }
+
+func (c Config) handleSelinux(options common.TranslateOptions) (types.Config, translate.TranslationSet, report.Report) {
+	rendered := types.Config{}
+	ts := translate.NewTranslationSet("yaml", "json")
+	var r report.Report
+	yamlPath := path.New("yaml", "selinux", "module")
+
+	hasValidModule := false
+
+	for _, module := range c.Selinux.Module {
+		if module.Name != "" && module.Content == "" || module.Content != "" && module.Name == "" || module.Name == "" && module.Content == "" {
+			r.AddOnWarn(path.New("yaml", "selinux", "module"), common.ErrFieldInvalid)
+		} else {
+			hasValidModule = true
+			break
+		}
+
+		if hasValidModule {
+			rendered.Storage.Filesystems = append(rendered.Storage.Filesystems,
+				types.Filesystem{
+					Device: "/dev/disk/by-label/boot",
+					Format: util.StrToPtr("ext4"),
+					Path:   util.StrToPtr("/boot"),
+				})
+
+			src, compression, err := baseutil.MakeDataURL([]byte(module.Content), nil, !options.NoResourceAutoCompression)
+			if err != nil {
+				r.AddOnError(yamlPath, err)
+				return rendered, ts, r
+			}
+
+			if module.Name != "" {
```

Comment:
If we have already validated this I dont think we need to check again. maybe we could make a list or array of valid selinux modules when we validate them?